pkgs/games/scummvm/default.nix

```diff
@@ -1,15 +1,17 @@
-{ lib, stdenv, fetchurl, nasm
-, alsa-lib, curl, flac, fluidsynth, freetype, libjpeg, libmad, libmpeg2, libogg, libvorbis, libGLU, libGL, SDL2, zlib
+{ lib, stdenv, fetchFromGitHub, nasm
+, alsa-lib, curl, flac, fluidsynth, freetype, libjpeg, libmad, libmpeg2, libogg, libtheora, libvorbis, libGLU, libGL, SDL2, zlib
 , Cocoa, AudioToolbox, Carbon, CoreMIDI, AudioUnit, cctools
 }:
 
 stdenv.mkDerivation rec {
   pname = "scummvm";
-  version = "2.5.1";
+  version = "2.6.1";
 
-  src = fetchurl {
-    url = "http://scummvm.org/frs/scummvm/${version}/${pname}-${version}.tar.xz";
-    sha256 = "sha256-n9jbOORFYUS/jDTazffyBOdfGOjkSOwBzgjOgmoDXwE=";
+  src = fetchFromGitHub {
+    owner = "${pname}";
```

Comment:
Two things here:
1. you don't have to interpolate - you could just reference it directly: `owner = pname;`, *but*
2. the fact the owner and repo happen to match the pname is a coincidence - the owner isn't a product of `pname`, so please just hardcode both like this `owner = "scummvm";`

Reply:
Fixed. Thanks for the review!

Reply:
I'd do the `repo` as well for the same reason. Additionally, we really should be using `hash` instead of `sha256`, so this is what I would make it look like:
```nix
  src = fetchFromGitHub {
    owner = "scummvm";
    repo = "scummvm";
    rev = "v${version}";
    hash = "sha256-fqMMdHBVcXLsBDWxXH9UKXwfvlyIVbRsIPmrYqPGQ+g=";
  };
```

Reply:
Fixed again. May I ask why `repo = pname` is not preferred? I see the use of this idiom throughout all of nixpkgs. Surely the repo name _is_ a product of `pname`? Just asking so I get it right next time.